client/src/app/project-registration/accession-field-group/accession-field-group.component.spec.ts

```diff
@@ -164,6 +181,18 @@ describe('AccessionFieldGroupComponent', () => {
       expect(insdcStudyCtrl.value).toEqual([]);
       expect(geoSeriesCtrl.value).toEqual([]);
       expect(bioStudiesCtrl.value).toEqual([]);
+      expect(egaCtrl.value).toEqual([]);
+    });
+
+    it('should set the EGA accession field', () => {
```

Comment:
This test suite seems very repetitive. The code for each accession type is similar. Can you think of a way to make it DRY?

Reply:
btw, unit tests could also be DAMP for readability.